server/package.json

```diff
@@ -21,7 +21,11 @@
     "jsonwebtoken": "^8.5.1",
     "morgan": "~1.9.1",
     "nodemon": "^1.19.1",
-    "puppeteer": "^2.1.1"
+    "pg": "^8.0.2",
+    "pg-hstore": "^2.3.3",
+    "puppeteer": "^2.1.1",
+    "sequelize": "^5.21.6",
+    "sequelize-cli": "^5.5.1"
```

Comment:
This one should be a dev dependency

Reply:
So, I may just be misunderstanding the difference between dependency and dev dependency, but aren't all of these also going to be used "out of development?" 
i.e. we need to be using all these libraries in the final product as well?

Reply:
I meant the cli only